src/Components/ReplyForm/ReplyForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import './ReplyForm.scss';
+import sendIcon from '../../assets/send.svg';
+
+interface IProps {
+  addReply: (newPost: string) => void;
+}
+
+interface IReply {
+  reply: string;
+  disabled: boolean;
+}
+
+class ReplyForm extends Component<IProps, IReply> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      reply: '',
+      disabled: true,
+    }
+  }
+
+  handleChange = (event: { target: { name: string; value: string; }; }) => {
+    this.setState(prevState => ({
+      ...prevState,
+      [event.target.name]: event.target.value
+    }))
+  }
+
+  toggleButton = () => {
+    if (this.state.reply !== '') {
+      this.setState({ disabled: false });
+    } else {
+      this.setState({ disabled: true });
+    }
+  }
+
+  submitReply = (event: { preventDefault: () => void; }) => {
+    event.preventDefault();
+    this.props.addReply(this.state.reply);
+    this.clearInput();
+  }
+
+  clearInput = () => {
+    this.setState({ reply: '' });
+  }
```

Comment:
We will need to clear timestamp as well. Not sure about author since we only have one user logging in